tools.html

```diff
@@ -261,18 +222,21 @@ <h4 class="card-title mbr-fonts-style display-5"><a href="https://aquasecurity.g
                         </div>
                     </div>
                 </div>
-              
+                <div class="card">
+                    <div class="card-wrapper">
+
+                        <div class="card-box">
+                            <h4 class="card-title mbr-fonts-style display-5"><a href="https://github.com/trustification/trustification" class="text-primary" target="_blank">Trustification</a></h4>
+                            <p class="card-text mbr-fonts-style display-4">A collection of software that allow you to store bill of materials (SBOM), vulnerability information (VEX) for your organization and use that information to learn impact of vulnerabilities and dependency changes.</p>
```

Comment:
Does it only support the CSAF-VEX profile? Wasn't able to already retrieve CSAF files? If so, I would suggest the following:
```suggestion
                            <p class="card-text mbr-fonts-style display-4">A collection of software that allow you to store bill of materials (SBOM), vulnerability information (e.g. from CSAF VEX) for your organization and use that information to learn impact of vulnerabilities and dependency changes. It is capable of collection the CSAF files automatically.</p>